src/common/BrowserStorage.ts

```diff
@@ -397,6 +413,16 @@ class _BrowserStorage {
 	 * They are only separated by type, to make it easier to understand the downgrade process.
 	 */
 	async downgrade(version: number) {
+		if (version > 10 && this.currentVersion <= 10) {
+			console.log('Downgrading to v10...');
+			const values = await this.get('options');
+			const options = values.options;
+			if (options) {
+				delete options.services['crunchyroll'];
```

Comment:
Same here. On the downgrade, we should copy it to `options.services['crunchyroll-beta']` before deleting it.